addons/web_editor/static/lib/odoo-editor/test/spec/list.test.js

```diff
@@ -4434,7 +4421,9 @@ describe('List', () => {
                                 contentAfter: unformat(`
                                     <ol>
                                         <li>abc</li>
-                                        <li>[]def</li>
+                                    </ol>
+                                    <p>[]def</p>
+                                    <ol>
```

Comment:
This is another example where this matches Google Docs conceptually but resets the numbering. cc @Zinston

Reply:
Yeah same comment as for the other case.